docs/simple-application.md

```diff
@@ -392,8 +392,7 @@ var User = {
 	load: function(id) {
 		return m.request({
 			method: "GET",
-			url: "https://rem-rest-api.herokuapp.com/api/users/:id",
-			data: {id: id},
+			url: "https://rem-rest-api.herokuapp.com/api/users/" + id,
```

Comment:
Can the IDs have any non-alphanumeric, non-`/[-_.!~*'()]/` characters? Just checking whether an `encodeURIComponent` is necessary here.